mlflow/utils/process.py

```diff
@@ -36,6 +37,7 @@ def _exec_cmd(
     capture_output=True,
     synchronous=True,
     stream_output=False,
+    terminate_signals=None,
```

Comment:
Can a docstring be added to this method to alert other devs what this functionality is for (since it is very frequently used throughout MLflow's code base)

Reply:
Good point, done!